src/postgraphql/withPostGraphQLContext.ts

```diff
@@ -61,7 +73,11 @@ export default async function withPostGraphQLContext(
 
   // Run the function with a context object that can be passed through
   try {
-    const pgRole = await setupPgClientTransaction({
+    if (!pgTransactionSetup) {
+      pgTransactionSetup = setupPgClientTransaction
+    }
+
+    const pgRole = await pgTransactionSetup({
```

Comment:
I do not like overwriting `arguments`; can we change this to something more like:

```js
const setupPgClientTransaction = pgTransactionSetup || defaultSetupPgClientTransaction;
const pgRole = await setupPgClientTransaction({
```

(and rename the existing `setupPgClientTransaction` to `defaultSetupPgClientTransaction`)